roles/chronos/tasks/main.yml

```diff
@@ -70,6 +70,19 @@
   tags:
     - chronos
 
+- name: configure chronos to use log4j override
+  sudo: yes
+  lineinfile:
+    dest: /usr/lib/systemd/system/chronos.service
+    line: Environment=JAVA_OPTS=-Dlog4j.configuration="file:///etc/chronos/log4j.properties"
+    insertbefore: "^ExecStart="
+    state: present
+  notify:
+    - reload chronos
+    - restart chronos
```

Comment:
it changes the unit file, which means we need a `systemd daemon-reload`, and to actually pick up the option afterwards it needs to restart.